postr/twitter_postr.py

```diff
@@ -209,13 +213,43 @@ def stream_and_graph(self, hashtags: List[str]) -> None:
 
     def graph_blob(self) -> None:
         """ Graphs a blob file for twitter sentiment """
+
+        def max_score(scores: List[float]) -> Tuple[int, float]:
```

Comment:
delete this method and replace with the other one. Make the list input type and the output tuple's second type parameter a generic

Reply:
Resolved. I removed score calculations for twitter_postr.py. It made sense to find a global max for instagram_postr.py, as followers/time generally has up to a few peaks close to each other. However, for tweet polarity, scores were often in the extremes (i.e. -1 and 1), so providing an absolute max or min doesn't show much, as this happens very often.